src/components/Content/index.tsx

```diff
@@ -11,32 +11,35 @@ import { routes } from "constants/routes";
 import { FullPageLoad } from "components/Loading/FullPageLoad";
 import { useAuthStateContext } from "context/auth";
 import { useQuery } from "@apollo/react-hooks";
-import { GET_PROJECTS } from "gql/queries/get-projects";
-import { ProjectsQuery } from "gql/generated/types";
+import { GET_USER } from "gql/queries";
+import { GetUserQuery } from "gql/generated/types";
 import { PageLayout } from "components/styles/Layout";
 import { PageDoesNotExist } from "pages/404";
 import { ConfigurePatch } from "pages/ConfigurePatch";
 import { Preferences } from "pages/Preferences";
+import { MyPatches } from "pages/MyPatches";
 
 export const Content: React.FC = () => {
   const { isAuthenticated, initialLoad } = useAuthStateContext();
 
   // this top-level query is required for authentication to work
   // afterware is used at apollo link level to authenticate or deauthenticate user based on response to query
   // therefore this could be any query as long as it is top-level
-  useQuery<ProjectsQuery>(GET_PROJECTS);
+  useQuery<GetUserQuery>(GET_USER);
```

Comment:
this query must complete for the app to render at all. if success, show page. if error, show error, and if not authenticated, show login. therefore waiting for it to complete does not change how long it takes for the page to load.

Reply:
it must complete but it doesn't have to be successful in order to mount the routes. the private routes redirect the user to the login page and they are mounted whether this query fails or not.